experiment-rollout.js

```diff
@@ -46,33 +46,53 @@ const data = {
         priority: [
             {
                 status: 2,
-                name: 'Brands'
+                name: 'Brands / Large bots | Reserved unless Discord partner for brands'
             },
             {
                 status: 2,
-                name: 'Discord Staff'
+                name: 'Discord Staff | Staff and personal accounts'
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Partnered / verified / ex-partnered / server subscription server owners'
             },
             {
                 status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                name: 'Verified bots | Ordered by guild count'
             },
             {
                 status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
-            },
```

Comment:
where did this go?

Reply:
Its unconfirmed, so I thought it was unnecessary? People take that stuff as face value and spread it as if it were 100% confirmed by Discord staff.

Reply:
yeah keeping it untill 100% confirmation will be better